src/manual.ts

```diff
@@ -11,12 +12,18 @@ else {
 
     // Define import file path.
     const inputFile = process.env.INPUT_FILE;
+    const outputFolder = process.env.E2G_OUTPUT_FOLDER || "tmp/e2g-output";
```

Comment:
I would make this just `e2g-output` (without the `tmp` for the manual runner. No need to put it two folders deep. Probably also add that folder to the gitignore